### ISSUE TITLE
How to resume training

### ISSUE THREAD

**Issue by GilLevi** [facebookresearch/InferSent#25]
Hi, 
First, thank you for releasing the code for your encoder and for senteval:)

I tried to add an option for resuming training, by adding the following code in line 122: 
nli_net = torch.load(os.path.join(params.outputdir, params.outputmodelname))

However, the loss and accuracy still start from random guess (about 33%) so I assume it didn't properly load the weights. What am I missing here? I'm pretty new to pytorch so sorry for the trivial question.

Thanks!
Gil

**Comment by aconneau** [comment]
Did you torch.load the whole model (I am asking because at the very end, we actually only save the encoder)?

**Comment by aconneau** [comment]
See https://github.com/facebookresearch/InferSent/commit/bcc794990d40774698ab492d29396721ae0b2f9c and https://github.com/facebookresearch/InferSent/issues/26

**Comment by aconneau** [comment]
Please re-open if problem not fixed. Thanks.
